### ISSUE TITLE
[Awaiting payment][$125] Web - Workspace - Console error shows up when clicking Share button

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#40086]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.62-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to Profile > Workspaces > any workspace.
3. Go to Profile.
4. Click Share.

## Expected Result:
No console error will show up.

## Actual Result:
Console error shows up.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/79236210-30ca-4648-9d58-d248941351dc

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01e680ef6d1ff9d76b</li>
        <li>Upwork Job ID: 1782410601436565504</li>
        <li>Last Price Increase: 2024-04-22</li>
<li>Automatic offers: </li>
<ul>
<li>shubham1206agra | Reviewer | 0</li>
<li>tienifr | Contributor | 0</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @johncschuster</details>

**Comment by izarutskaya** [comment]
@johncschuster I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #collect project.

**Comment by izarutskaya** [comment]
Production 


https://github.com/Expensify/App/assets/115492554/41482638-b55b-4289-9fb3-e1b9b07d0895

**Comment by pecanoro** [comment]
@tienifr Please, avoid posting proposals until the `Help Wanted` label since we tend to have original authors fix regressions.

**Comment by Julesssss** [comment]
Console log isn't a blocker, we have a separate issue [here](https://github.com/Expensify/App/issues/40080) for not being able to download the QR code.

**Comment by Julesssss** [comment]
Assuming the fix for the linked issue doesn't fix this, then we'd still need a fix. But lets wait to see what happens with https://github.com/Expensify/App/issues/40080

**Comment by pecanoro** [comment]
Still waiting on https://github.com/Expensify/App/issues/40080

**Comment by pecanoro** [comment]
It seems we ended up disabling the QR code for downloading the app but we still need to fix this problem. I am going to open it up for proposal review but since it's a super small issue, only a console error, I am going to set the price to $125.

**Comment by pecanoro** [comment]
Btw, the error seems slightly different now:

```
Error: <rect> attribute rx: Expected length, "NaN".
```

**Comment by johncschuster** [comment]
Bumping for Melvin

**Comment by pecanoro** [comment]
Just assigned @tienifr to the issue!

**Comment by trjExpensify** [comment]
I think you meant to tag @johncschuster as the BZ on this issue. 👍

**Comment by johncschuster** [comment]
Looking now

**Comment by johncschuster** [comment]
@shubham1206agra it looks like you haven't accepted the invite to the [job on Upwork](https://www.upwork.com/nx/wm/offer/102023926). Can you please accept that?

**Comment by johncschuster** [comment]
I've issued payment to @tienifr 🎉

**Comment by johncschuster** [comment]
Payment has been issued to @shubham1206agra. Thanks for your contributions!
